docs/ui-test.en-US.md

```diff
@@ -0,0 +1,157 @@
+---
+order: 15
+title: UI Test
+type: 进阶
+---
+
+UI testing is an important part of the project development process. Effective test cases can sort out business needs and ensure the quality and progress of development, It make engineers can refactor the code and add new features without trepidation.
+
+Ant Design Pro comes packaged with a handy set of easy-to-use React unit tests and E2E test framework to run test cases by running the following command at the project root.
+
+```bash
+$ npm run test:all  # Execute all the tests
+```
+
+<img src="https://gw.alipayobjects.com/zos/rmsportal/bNkhdMosBxuEhhKgnROo.png" width="700" />
+
+Now we start a quick introduction to writing your business test cases in your project.
+
+## Unit Tests
+
+Unit tests are used to test the appearance of React UI components。We referenced [create-react-app](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#running-tests), using [jest](http://facebook.github.io/jest/) as a test framework.
+
+jest is a node-side test framework that uses jsdom to simulate the DOM environment and is suitable for quick testing of the logical behavior of React components. If requiring a real browser, please refer to the E2E test section.
+
+### Write a case
+
+For example, we could create a file `src/routes/Result/Success.test.js` to test the UI appearance of successful page components.
+
+```jsx
+import React from 'react';
+import { shallow } from 'enzyme';
+import Success from './Success';   // Introduce the corresponding React component
+
+it('renders with Result', () => {
+  const wrapper = shallow(<Success />);                           // Rendering
+  expect(wrapper.find('Result').length).toBe(1);                  // Has Result component
+  expect(wrapper.find('Result').prop('type')).toBe('success');    // The type of the Result component is success
+});
+```
+
+Here [enzyme](http://airbnb.io/enzyme/docs/api/index.html) is used as a test library, which provides a number of useful APIs to help us test React components. Assert part use the jest acquiescence [jasmine2 expect grammar](https://facebook.github.io/jest/docs/en/expect.html#content).
+
+### Local execution
+
+Use the following command will be search and execute `* .test.js` format test case file in `src`.
```

Comment:
Files with suffix `* .test.js` in `src` will be found out and executed by using the following command: